src/realm/object_converter.cpp

```diff
@@ -284,35 +284,33 @@ void InterRealmValueConverter::copy_value(const Obj& src_obj, Obj& dst_obj, bool
 // If an embedded object is encountered, add it to a list of embedded objects to process.
 // This relies on the property that embedded objects only have one incoming link
 // otherwise there could be an infinite loop while discovering embedded objects.
-void EmbeddedObjectConverter::track(Obj e_src, Obj e_dst)
+void EmbeddedObjectConverter::track(const Obj& e_src, const Obj& e_dst)
 {
     embedded_pending.push_back({e_src, e_dst});
 }
 
 void EmbeddedObjectConverter::process_pending()
 {
-    // Conceptually this is a map, but doing a linear search through a vector is known
-    // to be faster for small number of elements. Since the number of tables expected
-    // to be processed here is assumed to be small < 20, use linear search instead of
-    // hashing. N is the depth to which embedded objects are connected and the upper
-    // bound is the total number of tables which is finite, and is usually small.
```

Comment:
This comment was written when the converters† were stored in a vector and isn't applicable now that they're stored in a map

† I am very confused by the things in this file being named "Converters".